Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -152,10 +152,25 @@
 
 void RemoteLayerTreeDrawingAreaProxy::commitLayerTree(IPC::Connection& connection, const Vector<std::pair<RemoteLayerTreeTransaction, RemoteScrollingCoordinatorTransaction>>& transactions)
 {
-    [CATransaction begin];
-    for (auto& transaction : transactions)
+    Vector<MachSendRight> sendRights;
+    for (auto& transaction : transactions) {
+        // commitLayerTreeTransaction consumes the incoming buffers, so we need to grab them first.
+        for (auto& [layerID, properties] : transaction.first.changedLayerProperties()) {
```

Comment:
Can we use an existing traversal to gather them, rather than adding a new one?

Reply:
I don't think it's worth smearing this touches-a-single-function patch across 4 classes for that, but maybe? I'm going to run with this one for now to fix the regression and we can consider making it messier later :)